dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/OpenAPIVersion.java

```diff
@@ -0,0 +1,144 @@
+/*******************************************************************************
+ * Copyright (c) 2024, 2025 IBM Corporation and others.
```

Comment:
Given this is a new file should it have 2024?

Reply:
It would, but it's not new, it was extracted from `OpenAPIVersionConfigImpl`.

Reply:
It used to be internal to that class, but I needed to use it in the new `OpenAPIVersionUsageChecker` so I moved it into its own class.